onto-viewer-web-app/src/test/java/configuration/ConfigLoaderTest.java

```diff
@@ -0,0 +1,189 @@
+package configuration;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Set;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.loader.ConfigLoader;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.loader.saxparser.ViewerCoreConfigurationHandler;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.ConfigItem;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.ConfigKeys;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.ViewerCoreConfiguration;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.element.DefaultLabelItem;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.element.LabelPriority;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.element.MissingLanguageItem;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.model.impl.element.StringItem;
+import org.edmcouncil.spec.ontoviewer.configloader.configuration.properties.AppProperties;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.jupiter.api.Assertions;
+import static org.junit.jupiter.api.Assertions.fail;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Patrycja Miazek (patrycja.miazek@makolab.com)
+ */
+public class ConfigLoaderTest {
+
+  private ViewerCoreConfiguration testViewerCoreConfig;
+
+  @BeforeClass
+  public static void setUpClass() {
+  }
+
+  @AfterClass
+  public static void tearDownClass() {
+  }
+
+  @Before
+  public void setUp() {
+
+    var configLoader = new ConfigLoader();
+    try {
+
+      Path path = Paths.get("integration_tests", "config");
```

Comment:
Loading configs only from one path for all tests limits things that you can test, i.e. you wouldn't test both cases: where a value is present, and where the same config is missing. IMO the best way to test this in the unit-testing manner would be to create (in some temporary directory) config files on the fly and then test them accordingly.